test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
@SethMichaelLarson There's no need to feel bad about not splitting the commit out. Things are overlooked all the time, and that's why we do code review. =) And remember: @sigmavirus24 and I have exactly this kind of experience on the other side of code review all the time (hit me up on Twitter and I'll point out my current OpenSSL code review saga).